Build failure: python3Packages.tornado

### Nixpkgs version

- Stable (25.05)

### Steps to reproduce

rebuilding system with march and mtune

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
❯ nix log /nix/store/f93q10c1yah4g0zxdm78gn9j7xcvnyrx-python3.12-tornado-6.5.1.drv
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/sf92gjjyh4vcxvmjlla672vip6lvwxk1-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/tox.ini"
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
/nix/store/q3pq2qk2csmc3m7hvyq7vclji2q79i33-python3.12-setuptools-78.1.1/lib/python3.12/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: Apache Software License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
running egg_info
creating tornado.egg-info
writing tornado.egg-info/PKG-INFO
writing dependency_links to tornado.egg-info/dependency_links.txt
writing top-level names to tornado.egg-info/top_level.txt
writing manifest file 'tornado.egg-info/SOURCES.txt'
reading manifest file 'tornado.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'docs/build'
warning: no files found matching 'tornado/test/README'
adding license file 'LICENSE'
writing manifest file 'tornado.egg-info/SOURCES.txt'
* Building wheel...
/nix/store/q3pq2qk2csmc3m7hvyq7vclji2q79i33-python3.12-setuptools-78.1.1/lib/python3.12/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/sf92gjjyh4vcxvmjlla672vip6lvwxk1-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/tox.ini"
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
/nix/store/q3pq2qk2csmc3m7hvyq7vclji2q79i33-python3.12-setuptools-78.1.1/lib/python3.12/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: Apache Software License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
running egg_info
creating tornado.egg-info
writing tornado.egg-info/PKG-INFO
writing dependency_links to tornado.egg-info/dependency_links.txt
writing top-level names to tornado.egg-info/top_level.txt
writing manifest file 'tornado.egg-info/SOURCES.txt'
reading manifest file 'tornado.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'docs/build'
warning: no files found matching 'tornado/test/README'
adding license file 'LICENSE'
writing manifest file 'tornado.egg-info/SOURCES.txt'
* Building wheel...
/nix/store/q3pq2qk2csmc3m7hvyq7vclji2q79i33-python3.12-setuptools-78.1.1/lib/python3.12/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: Apache Software License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
running bdist_wheel
running build
running build_py
creating build/lib.linux-x86_64-cpython-312/tornado
copying tornado/__init__.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/_locale_data.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/auth.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/autoreload.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/concurrent.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/curl_httpclient.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/escape.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/gen.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/http1connection.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/httpclient.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/httpserver.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/httputil.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/ioloop.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/iostream.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/locale.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/locks.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/log.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/netutil.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/options.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/process.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/queues.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/routing.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/simple_httpclient.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/tcpclient.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/tcpserver.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/template.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/testing.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/util.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/web.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/websocket.py -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/wsgi.py -> build/lib.linux-x86_64-cpython-312/tornado
creating build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/__init__.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/__main__.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/asyncio_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/auth_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/autoreload_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/circlerefs_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/concurrent_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/curl_httpclient_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/escape_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/gen_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/http1connection_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/httpclient_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/httpserver_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/httputil_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/import_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/ioloop_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/iostream_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/locale_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/locks_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/log_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/netutil_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/options_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/process_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/queues_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/resolve_test_helper.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/routing_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/runtests.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/simple_httpclient_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/tcpclient_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/tcpserver_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/template_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/testing_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/twisted_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/util.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/util_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/web_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/websocket_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/wsgi_test.py -> build/lib.linux-x86_64-cpython-312/tornado/test
creating build/lib.linux-x86_64-cpython-312/tornado/platform
copying tornado/platform/__init__.py -> build/lib.linux-x86_64-cpython-312/tornado/platform
copying tornado/platform/asyncio.py -> build/lib.linux-x86_64-cpython-312/tornado/platform
copying tornado/platform/caresresolver.py -> build/lib.linux-x86_64-cpython-312/tornado/platform
copying tornado/platform/twisted.py -> build/lib.linux-x86_64-cpython-312/tornado/platform
copying tornado/__init__.pyi -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/speedups.pyi -> build/lib.linux-x86_64-cpython-312/tornado
copying tornado/py.typed -> build/lib.linux-x86_64-cpython-312/tornado
creating build/lib.linux-x86_64-cpython-312/tornado/test/csv_translations
copying tornado/test/csv_translations/fr_FR.csv -> build/lib.linux-x86_64-cpython-312/tornado/test/csv_translations
creating build/lib.linux-x86_64-cpython-312/tornado/test/gettext_translations/fr_FR/LC_MESSAGES
copying tornado/test/gettext_translations/fr_FR/LC_MESSAGES/tornado_test.mo -> build/lib.linux-x86_64-cpython-312/tornado/test/gettext_translations/fr_FR/LC_MESSAGES
copying tornado/test/gettext_translations/fr_FR/LC_MESSAGES/tornado_test.po -> build/lib.linux-x86_64-cpython-312/tornado/test/gettext_translations/fr_FR/LC_MESSAGES
copying tornado/test/options_test.cfg -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/options_test_types.cfg -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/options_test_types_str.cfg -> build/lib.linux-x86_64-cpython-312/tornado/test
creating build/lib.linux-x86_64-cpython-312/tornado/test/static
copying tornado/test/static/robots.txt -> build/lib.linux-x86_64-cpython-312/tornado/test/static
copying tornado/test/static/sample.xml -> build/lib.linux-x86_64-cpython-312/tornado/test/static
copying tornado/test/static/sample.xml.gz -> build/lib.linux-x86_64-cpython-312/tornado/test/static
copying tornado/test/static/sample.xml.bz2 -> build/lib.linux-x86_64-cpython-312/tornado/test/static
creating build/lib.linux-x86_64-cpython-312/tornado/test/static/dir
copying tornado/test/static/dir/index.html -> build/lib.linux-x86_64-cpython-312/tornado/test/static/dir
copying tornado/test/static_foo.txt -> build/lib.linux-x86_64-cpython-312/tornado/test
creating build/lib.linux-x86_64-cpython-312/tornado/test/templates
copying tornado/test/templates/utf8.html -> build/lib.linux-x86_64-cpython-312/tornado/test/templates
copying tornado/test/test.crt -> build/lib.linux-x86_64-cpython-312/tornado/test
copying tornado/test/test.key -> build/lib.linux-x86_64-cpython-312/tornado/test
running build_ext
building 'tornado.speedups' extension
creating build/temp.linux-x86_64-cpython-312/tornado
gcc -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -I/nix/store/78s307i15px7a2b3hy0k7l9ygi206qn0-libxcrypt-4.4.38/include -fPIC -DPy_LIMITED_API=0x03090000 -I/nix/store/rqlj652nb7piqmrrpvnf2xb6ii756nda-python3-3.12.11/includ>
gcc -shared -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-bzip2-1.0.8/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-expat-2.7.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-libffi-3.4.8/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeee>
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/tornado
copying build/lib.linux-x86_64-cpython-312/tornado/__init__.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/_locale_data.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/auth.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/autoreload.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/concurrent.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/curl_httpclient.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/escape.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/gen.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/http1connection.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/httpclient.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/httpserver.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/httputil.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/ioloop.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/iostream.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/locale.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/locks.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/log.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/netutil.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/options.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/process.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/queues.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/routing.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/simple_httpclient.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/tcpclient.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/tcpserver.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/template.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/testing.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/util.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/web.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/websocket.py -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/wsgi.py -> build/bdist.linux-x86_64/wheel/./tornado
creating build/bdist.linux-x86_64/wheel/tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/__init__.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/__main__.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/asyncio_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/auth_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/autoreload_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/circlerefs_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/concurrent_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/curl_httpclient_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/escape_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/gen_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/http1connection_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/httpclient_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/httpserver_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/httputil_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/import_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/ioloop_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/iostream_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/locale_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/locks_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/log_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/netutil_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/options_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/process_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/queues_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/resolve_test_helper.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/routing_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/runtests.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/simple_httpclient_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/tcpclient_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/tcpserver_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/template_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/testing_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/twisted_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/util.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/util_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/web_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/websocket_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/wsgi_test.py -> build/bdist.linux-x86_64/wheel/./tornado/test
creating build/bdist.linux-x86_64/wheel/tornado/test/csv_translations
copying build/lib.linux-x86_64-cpython-312/tornado/test/csv_translations/fr_FR.csv -> build/bdist.linux-x86_64/wheel/./tornado/test/csv_translations
creating build/bdist.linux-x86_64/wheel/tornado/test/gettext_translations
creating build/bdist.linux-x86_64/wheel/tornado/test/gettext_translations/fr_FR
creating build/bdist.linux-x86_64/wheel/tornado/test/gettext_translations/fr_FR/LC_MESSAGES
copying build/lib.linux-x86_64-cpython-312/tornado/test/gettext_translations/fr_FR/LC_MESSAGES/tornado_test.mo -> build/bdist.linux-x86_64/wheel/./tornado/test/gettext_translations/fr_FR/LC_MESSAGES
copying build/lib.linux-x86_64-cpython-312/tornado/test/gettext_translations/fr_FR/LC_MESSAGES/tornado_test.po -> build/bdist.linux-x86_64/wheel/./tornado/test/gettext_translations/fr_FR/LC_MESSAGES
copying build/lib.linux-x86_64-cpython-312/tornado/test/options_test.cfg -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/options_test_types.cfg -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/options_test_types_str.cfg -> build/bdist.linux-x86_64/wheel/./tornado/test
creating build/bdist.linux-x86_64/wheel/tornado/test/static
copying build/lib.linux-x86_64-cpython-312/tornado/test/static/robots.txt -> build/bdist.linux-x86_64/wheel/./tornado/test/static
copying build/lib.linux-x86_64-cpython-312/tornado/test/static/sample.xml -> build/bdist.linux-x86_64/wheel/./tornado/test/static
copying build/lib.linux-x86_64-cpython-312/tornado/test/static/sample.xml.gz -> build/bdist.linux-x86_64/wheel/./tornado/test/static
copying build/lib.linux-x86_64-cpython-312/tornado/test/static/sample.xml.bz2 -> build/bdist.linux-x86_64/wheel/./tornado/test/static
creating build/bdist.linux-x86_64/wheel/tornado/test/static/dir
copying build/lib.linux-x86_64-cpython-312/tornado/test/static/dir/index.html -> build/bdist.linux-x86_64/wheel/./tornado/test/static/dir
copying build/lib.linux-x86_64-cpython-312/tornado/test/static_foo.txt -> build/bdist.linux-x86_64/wheel/./tornado/test
creating build/bdist.linux-x86_64/wheel/tornado/test/templates
copying build/lib.linux-x86_64-cpython-312/tornado/test/templates/utf8.html -> build/bdist.linux-x86_64/wheel/./tornado/test/templates
copying build/lib.linux-x86_64-cpython-312/tornado/test/test.crt -> build/bdist.linux-x86_64/wheel/./tornado/test
copying build/lib.linux-x86_64-cpython-312/tornado/test/test.key -> build/bdist.linux-x86_64/wheel/./tornado/test
creating build/bdist.linux-x86_64/wheel/tornado/platform
copying build/lib.linux-x86_64-cpython-312/tornado/platform/__init__.py -> build/bdist.linux-x86_64/wheel/./tornado/platform
copying build/lib.linux-x86_64-cpython-312/tornado/platform/asyncio.py -> build/bdist.linux-x86_64/wheel/./tornado/platform
copying build/lib.linux-x86_64-cpython-312/tornado/platform/caresresolver.py -> build/bdist.linux-x86_64/wheel/./tornado/platform
copying build/lib.linux-x86_64-cpython-312/tornado/platform/twisted.py -> build/bdist.linux-x86_64/wheel/./tornado/platform
copying build/lib.linux-x86_64-cpython-312/tornado/__init__.pyi -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/speedups.pyi -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/py.typed -> build/bdist.linux-x86_64/wheel/./tornado
copying build/lib.linux-x86_64-cpython-312/tornado/speedups.abi3.so -> build/bdist.linux-x86_64/wheel/./tornado
running install_egg_info
running egg_info
writing tornado.egg-info/PKG-INFO
writing dependency_links to tornado.egg-info/dependency_links.txt
writing top-level names to tornado.egg-info/top_level.txt
reading manifest file 'tornado.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'docs/build'
warning: no files found matching 'tornado/test/README'
adding license file 'LICENSE'
writing manifest file 'tornado.egg-info/SOURCES.txt'
Copying tornado.egg-info to build/bdist.linux-x86_64/wheel/./tornado-6.5.1-py3.12.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/tornado-6.5.1.dist-info/WHEEL
creating '/build/source/dist/.tmp-h17cpeby/tornado-6.5.1-cp39-abi3-linux_x86_64.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'tornado/__init__.py'
adding 'tornado/__init__.pyi'
adding 'tornado/_locale_data.py'
adding 'tornado/auth.py'
adding 'tornado/autoreload.py'
adding 'tornado/concurrent.py'
adding 'tornado/curl_httpclient.py'
adding 'tornado/escape.py'
adding 'tornado/gen.py'
adding 'tornado/http1connection.py'
adding 'tornado/httpclient.py'
adding 'tornado/httpserver.py'
adding 'tornado/httputil.py'
adding 'tornado/ioloop.py'
adding 'tornado/iostream.py'
adding 'tornado/locale.py'
adding 'tornado/locks.py'
adding 'tornado/log.py'
adding 'tornado/netutil.py'
adding 'tornado/options.py'
adding 'tornado/process.py'
adding 'tornado/py.typed'
adding 'tornado/queues.py'
adding 'tornado/routing.py'
adding 'tornado/simple_httpclient.py'
adding 'tornado/speedups.abi3.so'
adding 'tornado/speedups.pyi'
adding 'tornado/tcpclient.py'
adding 'tornado/tcpserver.py'
adding 'tornado/template.py'
adding 'tornado/testing.py'
adding 'tornado/util.py'
adding 'tornado/web.py'
adding 'tornado/websocket.py'
adding 'tornado/wsgi.py'
adding 'tornado/platform/__init__.py'
adding 'tornado/platform/asyncio.py'
adding 'tornado/platform/caresresolver.py'
adding 'tornado/platform/twisted.py'
adding 'tornado/test/__init__.py'
adding 'tornado/test/__main__.py'
adding 'tornado/test/asyncio_test.py'
adding 'tornado/test/auth_test.py'
adding 'tornado/test/autoreload_test.py'
adding 'tornado/test/circlerefs_test.py'
adding 'tornado/test/concurrent_test.py'
adding 'tornado/test/curl_httpclient_test.py'
adding 'tornado/test/escape_test.py'
adding 'tornado/test/gen_test.py'
adding 'tornado/test/http1connection_test.py'
adding 'tornado/test/httpclient_test.py'
adding 'tornado/test/httpserver_test.py'
adding 'tornado/test/httputil_test.py'
adding 'tornado/test/import_test.py'
adding 'tornado/test/ioloop_test.py'
adding 'tornado/test/iostream_test.py'
adding 'tornado/test/locale_test.py'
adding 'tornado/test/locks_test.py'
adding 'tornado/test/log_test.py'
adding 'tornado/test/netutil_test.py'
adding 'tornado/test/options_test.cfg'
adding 'tornado/test/options_test.py'
adding 'tornado/test/options_test_types.cfg'
adding 'tornado/test/options_test_types_str.cfg'
adding 'tornado/test/process_test.py'
adding 'tornado/test/queues_test.py'
adding 'tornado/test/resolve_test_helper.py'
adding 'tornado/test/routing_test.py'
adding 'tornado/test/runtests.py'
adding 'tornado/test/simple_httpclient_test.py'
adding 'tornado/test/static_foo.txt'
adding 'tornado/test/tcpclient_test.py'
adding 'tornado/test/tcpserver_test.py'
adding 'tornado/test/template_test.py'
adding 'tornado/test/test.crt'
adding 'tornado/test/test.key'
adding 'tornado/test/testing_test.py'
adding 'tornado/test/twisted_test.py'
adding 'tornado/test/util.py'
adding 'tornado/test/util_test.py'
adding 'tornado/test/web_test.py'
adding 'tornado/test/websocket_test.py'
adding 'tornado/test/wsgi_test.py'
adding 'tornado/test/csv_translations/fr_FR.csv'
adding 'tornado/test/gettext_translations/fr_FR/LC_MESSAGES/tornado_test.mo'
adding 'tornado/test/gettext_translations/fr_FR/LC_MESSAGES/tornado_test.po'
adding 'tornado/test/static/robots.txt'
adding 'tornado/test/static/sample.xml'
adding 'tornado/test/static/sample.xml.bz2'
adding 'tornado/test/static/sample.xml.gz'
adding 'tornado/test/static/dir/index.html'
adding 'tornado/test/templates/utf8.html'
adding 'tornado-6.5.1.dist-info/licenses/LICENSE'
adding 'tornado-6.5.1.dist-info/METADATA'
adding 'tornado-6.5.1.dist-info/WHEEL'
adding 'tornado-6.5.1.dist-info/top_level.txt'
adding 'tornado-6.5.1.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Successfully built tornado-6.5.1-cp39-abi3-linux_x86_64.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for tornado-6.5.1-cp39-abi3-linux_x86_64.whl
Finished executing pythonRuntimeDepsCheck
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Executing pypaInstallPhase
Successfully installed tornado-6.5.1-cp39-abi3-linux_x86_64.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: fixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
shrinking RPATHs of ELF executables and libraries in /nix/store/l40xwyz9vaw8g9ki9qhmh1xl8jjwl46v-python3.12-tornado-6.5.1
shrinking /nix/store/l40xwyz9vaw8g9ki9qhmh1xl8jjwl46v-python3.12-tornado-6.5.1/lib/python3.12/site-packages/tornado/speedups.abi3.so
checking for references to /build/ in /nix/store/l40xwyz9vaw8g9ki9qhmh1xl8jjwl46v-python3.12-tornado-6.5.1...
patching script interpreter paths in /nix/store/l40xwyz9vaw8g9ki9qhmh1xl8jjwl46v-python3.12-tornado-6.5.1
stripping (with command strip and flags -S -p) in  /nix/store/l40xwyz9vaw8g9ki9qhmh1xl8jjwl46v-python3.12-tornado-6.5.1/lib
shrinking RPATHs of ELF executables and libraries in /nix/store/qzisq3hlmvwqzn3khb3msc0blnb7wd0b-python3.12-tornado-6.5.1-dist
checking for references to /build/ in /nix/store/qzisq3hlmvwqzn3khb3msc0blnb7wd0b-python3.12-tornado-6.5.1-dist...
patching script interpreter paths in /nix/store/qzisq3hlmvwqzn3khb3msc0blnb7wd0b-python3.12-tornado-6.5.1-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: tornado
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
pytest flags: -m pytest --ignore-glob=maint/test --ignore-glob=tornado/test/iostream_test.py -k not\ \(test_unix_socket_bad_request\)
============================= test session starts ==============================
platform linux -- Python 3.12.11, pytest-8.3.5, pluggy-1.5.0
rootdir: /build/source
configfile: pyproject.toml
collecting ... ^Mcollecting 0 items                                                             ^Mcollecting 0 items                                                             ^Mcollecting 108 items                                    >

tornado/test/asyncio_test.py .............                               [  1%]
tornado/test/auth_test.py ...................                            [  2%]
tornado/test/autoreload_test.py ....                                     [  3%]
tornado/test/circlerefs_test.py ......                                   [  3%]
tornado/test/concurrent_test.py ...........                              [  4%]
tornado/test/curl_httpclient_test.py sssssssssssssssssssssssssssssssssss [  8%]
sssssssss                                                                [  8%]
tornado/test/escape_test.py ...........                                  [  9%]
tornado/test/gen_test.py ............................................... [ 14%]
..................F......                                                [ 16%]
tornado/test/http1connection_test.py .                                   [ 16%]
tornado/test/httpclient_test.py ........................................ [ 20%]
....................                                                     [ 21%]
tornado/test/httpserver_test.py ................................s....... [ 25%]
...............................                                          [ 28%]
tornado/test/httputil_test.py .......................................... [ 32%]
..........                                                               [ 33%]
tornado/test/import_test.py ...                                          [ 33%]
tornado/test/ioloop_test.py ............................................ [ 37%]
..........                                                               [ 38%]
tornado/test/locale_test.py ........                                     [ 39%]
tornado/test/locks_test.py ........................................      [ 42%]
tornado/test/log_test.py ...............                                 [ 44%]
tornado/test/netutil_test.py ss.......s...                               [ 45%]
tornado/test/options_test.py ........................                    [ 47%]
tornado/test/process_test.py .........                                   [ 48%]
tornado/test/queues_test.py .........................................    [ 52%]
tornado/test/routing_test.py ......                                      [ 52%]
tornado/test/simple_httpclient_test.py ................................. [ 55%]
.......sssssssssssssssssssssssssss...................................... [ 62%]
..................................                                       [ 65%]
tornado/test/tcpclient_test.py ..ss...s......................            [ 67%]
tornado/test/tcpserver_test.py ........                                  [ 68%]
tornado/test/template_test.py ...............s.......................... [ 72%]
.                                                                        [ 72%]
tornado/test/testing_test.py ......s..............                       [ 74%]
tornado/test/twisted_test.py ss                                          [ 74%]
tornado/test/util_test.py ......................                         [ 76%]
tornado/test/web_test.py ............................................... [ 80%]
.....................................................................s.. [ 87%]
........................................................................ [ 94%]
....                                                                     [ 94%]
tornado/test/websocket_test.py ...................................ssss.. [ 98%]
......s.s.....                                                           [ 99%]
tornado/test/wsgi_test.py ......                                         [100%]

=================================== FAILURES ===================================
_____________________________ RunnerGCTest.test_gc _____________________________

self = <tornado.test.gen_test.RunnerGCTest testMethod=test_gc>

    @gen_test
    def test_gc(self):
        # GitHub issue 1769: Runner objects can get GCed unexpectedly
        # while their future is alive.
        weakref_scope = [None]  # type: List[Optional[weakref.ReferenceType]]
    
        def callback():
            gc.collect(2)
            weakref_scope[0]().set_result(123)  # type: ignore
    
        @gen.coroutine
        def tester():
            fut = Future()  # type: Future[int]
            weakref_scope[0] = weakref.ref(fut)
            self.io_loop.add_callback(callback)
            yield fut
    
>       yield gen.with_timeout(datetime.timedelta(seconds=0.2), tester())

tornado/test/gen_test.py:964: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tornado.gen.Runner object at 0x7ffff58b3740>

    def run(self) -> None:
        """Starts or resumes the generator, running until it reaches a
        yield point that is not ready.
        """
        if self.running or self.finished:
            return
        try:
tornado/test/log_test.py ...............                                 [ 44%]
tornado/test/netutil_test.py ss.......s...                               [ 45%]
tornado/test/options_test.py ........................                    [ 47%]
tornado/test/process_test.py .........                                   [ 48%]
tornado/test/queues_test.py .........................................    [ 52%]
tornado/test/routing_test.py ......                                      [ 52%]
tornado/test/simple_httpclient_test.py ................................. [ 55%]
.......sssssssssssssssssssssssssss...................................... [ 62%]
..................................                                       [ 65%]
tornado/test/tcpclient_test.py ..ss...s......................            [ 67%]
tornado/test/tcpserver_test.py ........                                  [ 68%]
tornado/test/template_test.py ...............s.......................... [ 72%]
.                                                                        [ 72%]
tornado/test/testing_test.py ......s..............                       [ 74%]
tornado/test/twisted_test.py ss                                          [ 74%]
tornado/test/util_test.py ......................                         [ 76%]
tornado/test/web_test.py ............................................... [ 80%]
.....................................................................s.. [ 87%]
........................................................................ [ 94%]
....                                                                     [ 94%]
tornado/test/websocket_test.py ...................................ssss.. [ 98%]
......s.s.....                                                           [ 99%]
tornado/test/wsgi_test.py ......                                         [100%]

=================================== FAILURES ===================================
_____________________________ RunnerGCTest.test_gc _____________________________

self = <tornado.test.gen_test.RunnerGCTest testMethod=test_gc>

    @gen_test
    def test_gc(self):
        # GitHub issue 1769: Runner objects can get GCed unexpectedly
        # while their future is alive.
        weakref_scope = [None]  # type: List[Optional[weakref.ReferenceType]]
    
        def callback():
            gc.collect(2)
            weakref_scope[0]().set_result(123)  # type: ignore
    
        @gen.coroutine
        def tester():
            fut = Future()  # type: Future[int]
            weakref_scope[0] = weakref.ref(fut)
            self.io_loop.add_callback(callback)
            yield fut
    
>       yield gen.with_timeout(datetime.timedelta(seconds=0.2), tester())

tornado/test/gen_test.py:964: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tornado.gen.Runner object at 0x7ffff58b3740>

    def run(self) -> None:
        """Starts or resumes the generator, running until it reaches a
        yield point that is not ready.
        """
        if self.running or self.finished:
            return
        try:
            self.running = True
            while True:
                future = self.future
                if future is None:
                    raise Exception("No pending future")
                if not future.done():
                    return
                self.future = None
                try:
                    try:
>                       value = future.result()
E                       TimeoutError: Timeout

tornado/gen.py:769: TimeoutError
=============================== warnings summary ===============================
tornado/test/tcpclient_test.py:40
  /build/source/tornado/test/tcpclient_test.py:40: PytestCollectionWarning: cannot collect test class 'TestTCPServer' because it has a __init__ constructor (from: tornado/test/tcpclient_test.py)
    class TestTCPServer(TCPServer):

tornado/test/util_test.py:43
  /build/source/tornado/test/util_test.py:43: PytestCollectionWarning: cannot collect test class 'TestConfigurable' because it has a __new__ constructor (from: tornado/test/util_test.py)
    class TestConfigurable(Configurable):

tornado/test/util_test.py:53
  /build/source/tornado/test/util_test.py:53: PytestCollectionWarning: cannot collect test class 'TestConfig1' because it has a __new__ constructor (from: tornado/test/util_test.py)
    class TestConfig1(TestConfigurable):

tornado/test/util_test.py:59
  /build/source/tornado/test/util_test.py:59: PytestCollectionWarning: cannot collect test class 'TestConfig2' because it has a __new__ constructor (from: tornado/test/util_test.py)
    class TestConfig2(TestConfigurable):

tornado/test/util_test.py:65
  /build/source/tornado/test/util_test.py:65: PytestCollectionWarning: cannot collect test class 'TestConfig3' because it has a __new__ constructor (from: tornado/test/util_test.py)
    class TestConfig3(TestConfigurable):

tornado/test/util_test.py:76
  /build/source/tornado/test/util_test.py:76: PytestCollectionWarning: cannot collect test class 'TestConfig3A' because it has a __new__ constructor (from: tornado/test/util_test.py)
    class TestConfig3A(TestConfig3):

tornado/test/util_test.py:81
  /build/source/tornado/test/util_test.py:81: PytestCollectionWarning: cannot collect test class 'TestConfig3B' because it has a __new__ constructor (from: tornado/test/util_test.py)
    class TestConfig3B(TestConfig3):

tornado/test/websocket_test.py:45
  /build/source/tornado/test/websocket_test.py:45: PytestCollectionWarning: cannot collect test class 'TestWebSocketHandler' because it has a __init__ constructor (from: tornado/test/websocket_test.py)
    class TestWebSocketHandler(WebSocketHandler):

tornado/test/process_test.py::ProcessTest::test_multi_process
tornado/test/process_test.py::ProcessTest::test_multi_process
tornado/test/process_test.py::ProcessTest::test_multi_process
tornado/test/process_test.py::ProcessTest::test_multi_process
tornado/test/process_test.py::ProcessTest::test_multi_process
tornado/test/process_test.py::ProcessTest::test_multi_process
  /build/source/tornado/process.py:128: DeprecationWarning: This process (pid=414) is multi-threaded, use of fork() may lead to deadlocks in the child.
    pid = os.fork()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tornado/test/gen_test.py::RunnerGCTest::test_gc - TimeoutError: Timeout
= 1 failed, 1008 passed, 89 skipped, 3 deselected, 14 warnings in 69.01s (0:01:09) =
```

### Additional context

_No response_

### System metadata

```
❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.41, NixOS, 25.05 (Warbler), 25.05.20250815.ad7196a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - channels(root): `"nixos-25.05"`
 - nixpkgs: `/nix/store/fbnlawcsvls3033n9ddi81grgw030881-source`
```

### Notify maintainers

No maintainers

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/tornado/default.nix
+++ b/pkgs/development/python-modules/tornado/default.nix
@@ -20,14 +20,14 @@
 
 buildPythonPackage rec {
   pname = "tornado";
-  version = "6.5.1";
+  version = "6.5.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "tornadoweb";
     repo = "tornado";
     tag = "v${version}";
-    hash = "sha256-CtmIjPKxKC0T8PGQW1wIAJm/+XxMzZXVZyZxV56sZME=";
+    hash = "sha256-jy/HnMY459yZX3HW9V61/ZSSanCJEZakBU/2pocGc/s=";
   };
 
   build-system = [ setuptools ];
@@ -38,14 +38,6 @@ buildPythonPackage rec {
     # additional tests that have extra dependencies, run slowly, or produce more output than a simple pass/fail
     # https://github.com/tornadoweb/tornado/blob/v6.2.0/maint/test/README
     "maint/test"
-
-    # AttributeError: 'TestIOStreamWebMixin' object has no attribute 'io_loop'
-    "tornado/test/iostream_test.py"
-  ];
-
-  disabledTests = [
-    # Exception: did not get expected log message
-    "test_unix_socket_bad_request"
   ];
 
   pythonImportsCheck = [ "tornado" ];

--- a/pkgs/development/python-modules/tornado/default.nix
+++ b/pkgs/development/python-modules/tornado/default.nix
@@ -59,10 +59,10 @@ buildPythonPackage rec {
       ;
   };
 
-  meta = with lib; {
+  meta = {
     description = "Web framework and asynchronous networking library";
     homepage = "https://www.tornadoweb.org/";
-    license = licenses.asl20;
+    license = lib.licenses.asl20;
     maintainers = [ ];
   };
 }

